config/wpcalypso.json

```diff
@@ -125,7 +125,7 @@
 		"post-editor/checkout-overlay": true,
 		"post-list/qr-code-link": true,
 		"press-this": true,
-		"promote-post/redesign-i2": true,
+		"promote-post/redesign-i2": false,
```

Comment:
Any specific reason for turning this off? We were trying to use this flag to test the widget flow redesign as well. If this is intentional, then we'll need to figure out another way.

Reply:
@benzhu56, you can test the feature in the browser by appending `?flags=promote-post/redesign-i2` to the URL.
I don't know if that will help you with your testing; I imagine that you can't always add that flag in the URL (for example, in mobile apps).

Reply:
Sorry @benzhu56  I mentioned this to the self-serve team, but didn't realise it was dual purpose.  Please feel free to turn it back on now credits have shipped.

Thanks, @sbarbosa TIL!

Reply:
Thank you both! We'll try appending the flags in the URL first. If it doesn't work, then we'll turn it on.